docs/src/Table/index.js

```diff
@@ -117,16 +111,6 @@ class TableExample extends React.Component {
     this.forceUpdate();
```

Comment:
yeah this was unrelated to the transition i believe. this had something to do with the infinite scroll table and gemini scrollbar